### PR TITLE
Fix: addon timeout is not acturally tracked

### PR DESCRIPTION
**Pull request by chivalryq**
### Description of your changes

Now there is a bug: addon enable timeout won't quit the enable process.

image

This PR:
1. fix addon enable timeout is not used
2. better addon enable fail message 

Signed-off-by: qiaozp <chivalry.pp@gmail.com>

I have:

- [x] Read and followed KubeVela's [contribution process](https://github.com/oam-dev/kubevela/blob/master/contribute/create-pull-request.md).
- [ ] [Related Docs](https://github.com/oam-dev/kubevela.io) updated properly. In a new feature or configuration option, an update to the documentation is necessary. 
- [x] Run `make reviewable` to ensure this PR is ready for review.
- [x] Added `backport release-x.y` labels to auto-backport this PR if necessary.

### How has this code been tested

Test by build a binary with very short timeout, timeout fail like this:
image

### CODE REVIEW

**Review by wonderflow — APPROVED** [review]

File: references/cli/addon.go [hunk 1]
```diff
@@ -404,14 +404,14 @@ func (a *Addon) enable(ctx context.Context, k8sClient client.Client, name string
...
-		return errors.Wrap(err, "Error occurs when waiting addon applicatoin running")
+		return err
 	}
 	return nil
 }
 
 func waitApplicationRunning(obj *v1beta1.Application) error {
 	trackInterval := 5 * time.Second
-	timeout := 10 * time.Minute
+	timeout := 600 * time.Second
```
Comment (zzxwill) [inline references/cli/addon.go:414]: Any differences?

File: references/cli/addon.go [hunk 2]
```diff
@@ -431,6 +431,9 @@ func waitApplicationRunning(obj *v1beta1.Application) error {
 		timeConsumed := int(time.Since(start).Seconds())
 		applySpinnerNewSuffix(spinner, fmt.Sprintf("Waiting addon application running. It is now in phase: %s (timeout %d/%d seconds)...",
 			phase, timeConsumed, int(timeout.Seconds())))
+		if timeConsumed > int(timeout.Seconds()) {
+			return errors.Errorf("Enable timeout, please run \"vela status %s -n vela-system\" to check addon application status", obj.Name)
```
Comment (zzxwill) [inline references/cli/addon.go:435]:
```suggestion
			return errors.Errorf("Enabling timeout, please run \"vela status %s -n vela-system\" to check the status of the addon", obj.Name)
```

We don't need to emphasize an add-on id deployed through an application, which is internal implementation details.

**Review by Somefive — APPROVED** [review]

File: references/cli/addon.go (hunk 1 above)
Comment (chivalryq) [reply references/cli/addon.go:414]: They are equivalent

**Pull request merged**